test/test-get-passwd.c

```diff
@@ -45,9 +45,15 @@ TEST_IMPL(get_passwd) {
   ASSERT(len > 0);
 
 #ifdef _WIN32
-  ASSERT(pwd.homedir[len - 1] != '\\');
+  if (len == 3 && pwd.homedir[1] == ':')
```

Comment:
I don't think so, but I could be wrong. If it is possible, I think we have other problems. We're using this [logic](https://github.com/libuv/libuv/blob/d476185bb373601b884b9a75746b8e9f666a43ba/src/win/util.c#L172) in other places to strip trailing slashes.